main.py

```diff
@@ -1,31 +1,30 @@
 import argparse
-import sys
 import os
 import shutil
+import sys
 import time
 import warnings
 from random import sample
 
 import numpy as np
-from sklearn import metrics
 import torch
 import torch.nn as nn
 import torch.optim as optim
+from sklearn import metrics
 from torch.autograd import Variable
 from torch.optim.lr_scheduler import MultiStepLR
 
-from cgcnn.model import CrystalGraphConvNet
-from cgcnn.data import collate_pool, get_train_val_test_loader
 from cgcnn.data import CIFData
-
+from cgcnn.data import collate_pool, get_train_val_test_loader
+from cgcnn.model import CrystalGraphConvNet
 
 parser = argparse.ArgumentParser(description='Crystal Graph Convolutional Neural Networks')
 parser.add_argument('data_options', metavar='OPTIONS', nargs='+',
                     help='dataset options, started with the path to root dir, '
-                    'then other options')
+                         'then other options')
 parser.add_argument('--task', choices=['regression', 'classification'],
-                    default='regression', help='complete a regression or '
-                    'classification task (default: regression)')
+                    default='classification', help='complete a regression or '
```

Comment:
Why do you change the default to classification?

Reply:
It's just for debugging. If you think that the default should be regression, I'll change it back.

Reply:
Yes, the default should be regression because most people will use it. Please change it back.